public/docs/ts/latest/tutorial/toh-pt2.jade

```diff
@@ -3,21 +3,20 @@ include ../_util-fns
 :marked
   # It Takes Many Heroes
   Our story needs more heroes.
-  We’ll expand our Tour of Heroes app to display a list of heroes,
-  allow the user to select a hero, and display the hero’s details.
-
-  Run the <live-example></live-example> for this part.
-
-  Let’s take stock of what we’ll need to display a list of heroes.
-  First, we need a list of heroes. We want to display those heroes in the view’s template,
-  so we’ll need a way to do that.
+  In this page, you'll expand the Tour of Heroes app to display a list of heroes,
+  allow the user to select a hero, and display the hero's details.
 
+  **Try it out**. Here's a link to a <live-example></live-example>.
+  
+  
 .l-main-section
 :marked
-  ## Where We Left Off
-  Before we continue with Part 2 of the Tour of Heroes,
-  let’s verify we have the following structure after [Part 1](./toh-pt1.html).
-  If not, we’ll need to go back to Part 1 and figure out what we missed.
+  ## Where you left off
+  Before you continue with Part 2 of the Tour of Heroes,
+  verify that you have the following structure after [Part 1](./toh-pt1.html).
+  If not, you'll need to go back to Part 1 and figure out what you missed.
+// CF: In the left nav bar, "Part 1" is numbered "2" and "Part 2" is numbered "3,"
+ so referring to "Part 1" etc. could be confusing.
```

Comment:
Instead of referring to "part" numbers, would it be better to use "the previous page", "this page", etc? or the page name?